### ISSUE TITLE
Data Dashboard for Expedited EDA

### ISSUE THREAD

**Issue by egrace479** [Imageomics/Image-Datapalooza-2023#2]
There’s a lot of available data, but determining its level of usefulness (via exploratory data analysis) can be a very time-consuming process. I have been working on a [dashboard](https://github.com/Imageomics/dashboard-prototype) to visualize distribution information and samples of datasets efficiently and with no coding required. We have a [pre-release hosted](https://dash.imageomics.org/) for use during Image Datapalooza. It is currently a bit Imageomics-focused, though images aren’t required to gather distribution statistics. I think it would be great to expand out the functionality for a more general audience as a way to quickly generate visuals for data or explore whether new data would be suitable for experiments without having to invest large amounts of time into EDA.

**Comment by nickynicolson** [comment]
Could this integrate with the practice of documenting datasets via "dataset cards"? (see e.g. https://huggingface.co/docs/hub/datasets-cards):

**Comment by egrace479** [comment]
Definitely. I have actually used it to generate a static display for one of the Institute's datasets (currently private) and plan to do so for more. I would be interested in integrating the interactive dashboard into a [HF Space](https://huggingface.co/spaces/launch) as well.

**Comment by egrace479** [comment]
Turns out deployment in a HF Space is relatively straightforward with a Docker container. We have v1.0.0 on [HF](https://huggingface.co/spaces/imageomics/dashboard-prototype), and set up a [development version](https://huggingface.co/spaces/imageomics/dev-dashboard) for testing off our [dev branch](https://github.com/Imageomics/dashboard-prototype/tree/dev).
